### ISSUE TITLE
False TrailingSemicolon warning

### ISSUE THREAD

**Issue by bendera** [brigade/scss-lint#241]
The scss-lint report <code>TrailingSemicolon</code> warning for the following code, if the <code>StringQuotes</code> linter is disabled:

```
@charset "UTF-8";

.foo:before {
    content: '▼';
}

```
